pkg/sentry/server/server.go

```diff
@@ -0,0 +1,162 @@
+package server
+
+import (
+	"context"
+	"crypto/tls"
+	"fmt"
+	"net"
+	"time"
+
+	pb "github.com/dapr/dapr/pkg/proto/sentry"
+	"github.com/dapr/dapr/pkg/sentry/ca"
+	"github.com/dapr/dapr/pkg/sentry/certs"
+	"github.com/dapr/dapr/pkg/sentry/csr"
+	"github.com/golang/protobuf/ptypes"
+	log "github.com/sirupsen/logrus"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+const (
+	serverCertExpiryBuffer = time.Minute
```

Comment:
certexpirybuffer seems too small. in production, I used to set at least 15 mins for 1 hour expiry cert due to the clock skew.

Reply:
Ok

Reply:
changed